src/db.ts

```diff
@@ -54,7 +54,8 @@ export const migrate = async () => {
         "identify-mock691f4e49fc43b913bd8ede668e187e9a-1509032370615": {
           id: "identify-mock691f4e49fc43b913bd8ede668e187e9a-1509032370615",
           reference: null,
-          url: "https://go.test.idnow.de/kontist/identifications/identify-mock691f4e49fc43b913bd8ede668e187e9a-1509032370615",
+          url:
+            "https://go.test.idnow.de/kontist/identifications/identify-mock691f4e49fc43b913bd8ede668e187e9a-1509032370615",
```

Comment:
❓ What did changed here?

Reply:
It was automatically changed by linter/prettier 🤷‍♂️  
Probably it was missed by previous person who changed that file